test/forge/BaseTest.sol

```diff
@@ -330,13 +330,15 @@ contract BaseTest is Test {
     {
         Id _id = _marketParams.id();
 
-        uint256 collateral = morpho.collateral(_id, borrower);
         uint256 collateralPrice = IOracle(_marketParams.oracle).price();
-        uint256 maxRepaidAssets = morpho.expectedBorrowAssets(_marketParams, borrower);
+        uint256 borrowShares = morpho.borrowShares(_id, borrower);
+        (,, uint256 totalBorrowAssets, uint256 totalBorrowShares) = morpho.expectedMarketBalances(_marketParams);
+        uint256 maxRepaidAssets = borrowShares.toAssetsDown(totalBorrowAssets, totalBorrowShares);
         uint256 maxSeizedAssets = maxRepaidAssets.wMulDown(_liquidationIncentiveFactor(_marketParams.lltv)).mulDivDown(
             ORACLE_PRICE_SCALE, collateralPrice
         );
 
+        uint256 collateral = morpho.collateral(_id, borrower);
```

Comment:
nb: this could have stayed at the same place, but it's fine here too

Reply:
This was a hack to avoid stack too deep :)